README.md

```diff
@@ -60,7 +60,8 @@ Additional components:
 The subdirectories above may include READMEs with additional info.
 
 "make" is used to build the components, or "gmake" on BSD-based systems.
-You will also need bash, gcc, and Python (at least 2.7 or 3.3).
+You will also need bash, gcc, and Python 3.3+ (if your system only has Python 2.7
```

Comment:
Do most standard Python 3.3+ installations automatically have `python3` in the path? If not, or maybe even irregardless, this should probably explicitly be mentioned so that it's clear what is expected here.

Reply:
> Do most standard Python 3.3+ installations automatically have `python3` in the path?

Good point.  I'd guess they do, since PEP 394 is pretty clean about there being a `python2` if Python 2.x is installed, and a `python3` if Python 3.x is installed.  And since Python 3.x is newer it's more likely that `python3` exists as a command (rather than `python2` existing if Python 2.x is installed).

But yes the wording here would benefit from additional clarity about the requirement of a `python3` command.

Reply:
I pushed a change to this effect.